### ISSUE TITLE
Retire GeneInfo

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#693]
🤖 Written by Claude

`GeneInfo` is a gene-tagging model that puts small icons next to gene symbols on GeneGrid. It ships three rows (created in `genes/migrations/0002_initial_data.py`), each pointing at an optional uploaded `GeneList`:

* Alternative Haplotype
* Pseudogene
* Triplet Repeat Disorders

The original ask on this issue was to document what these are and how to install them. Having dug into it, all three categories are now either obsolete or better served by something we already have, so the resolution is to **retire GeneInfo** rather than document it.

## Why not just document it

The install process is the problem, not the missing docs:

1. Run the migration (creates the three rows with `gene_list = NULL`)
2. Obtain three text files from somewhere
3. Upload each as a gene list
4. Go into Django admin and hand-wire each `GeneInfo.gene_list`

Until step 4 is done the whole feature compiles out of the template — `gene_grid_tags.py:138` sets `has_gene_info` only if some `GeneInfo` has a gene list with symbols in it. So on any deployment where nobody did the manual wiring, the feature is invisible and has always been.

The source files came from SA Path diagnostics (`/tau/references/annotations/`, now `hg38_reference_files/gene_annotations/` in the GMP-TAU pipeline) — `Pseudogenes.txt`, `TripletRepeatDisorders.txt`, `all_alt_haps_with_a_transcript_ID_170623.txt`. They are internally curated, not public downloads. Original implementation was #1127 (2018).

### Triplet Repeat Disorders → already covered by PanelApp

PanelApp exposes repeat expansions as first-class STR entities, and they already surface on the gene page via `panel_app_gene_evidence` in `gene_symbol_info_tab.html`. Users who want to filter can also just add a PanelApp panel as a gene list.

PanelApp is strictly better than the file it replaces — the 2017 file had 30 gene symbols and nothing else:

| | STR entities | distinct genes | green (confidence 3) |
|---|---|---|---|
| PanelApp Australia | 264 | 71 | 55 |
| PanelApp England | 226 | 37 | 19 |

Each entity carries the motif, both GRCh37 and GRCh38 coordinates, normal/pathogenic repeat thresholds, mode of inheritance, OMIM phenotypes and PMIDs.

Worth noting the old name was wrong anyway — observed motif lengths in PanelApp AU are 3, 4, 5, 6, 10, 12, 13, 20, 24 and 99, so "triplet repeat" undersells it.

### Alternative Haplotype → obsolete

A GRCh37 artefact: the list was genes with a transcript on a `_hap` / `_random` / `Un_` contig, generated in-house by contig-name match. SA Path diagnostics dropped it entirely in their hg38 rewrite — the current `single_sample_exec_summary.py` has no alt-haplotype limitation section. Nothing to replace.

### Pseudogene → follow-up, see below

The one category with genuine ongoing clinical value, but the right source and the right shape are both different. Handled as a separate piece of work rather than a port.

## Work: delete GeneInfo

**Model / data**
* `genes/models.py:2059` — `GeneInfo` model
* `genes/models.py:1564` — `GeneListCategory.GENE_INFO` constant
* New migration to drop the model. Decide what to do with the `GeneInfo` `GeneListCategory` and any gene lists in it — on deployments that did the manual wiring those are real uploaded gene lists, so probably leave the data and just drop the constant.

**Admin**
* `genes/admin.py:54` — `GeneInfoAdmin`

**REST API** (breaking change — these are public endpoints, may need a note in `variantgrid_api`)
* `genes/urls.py:101` — `api_batch_gene_info`
* `genes/urls.py:103` — `api_gene_info`
* `genes/views/views_rest.py:217` — `GeneInfoView`
* `genes/views/views_rest.py:226` — `BatchGeneInfoView`
* `genes/serializers.py:237` — `GeneInfoSerializer`

**Views**
* `genes/views/views.py:311` — `gene_infos` cached property, and the `"gene_infos"` entry in the lazy context list at `:396`. Already dead — nothing renders it.

**GeneGrid template / JS**
* `genes/templatetags/gene_grid_tags.py:138,157` — `has_gene_info`
* `genes/templates/genes/tags/gene_grid_tag.html` — `geneInfoTimer` / `geneInfoRequestId` / `geneInfoData` / `geneInfoRequests` (`:206-209`), `getGeneInfo` (`:977`), `batchRetrieveGeneInfo` (`:998`), `setGeneInfo` (`:1027`), `handleBatchRetrieveGeneInfo` (`:1043`), call site at `:1118`

**Gene symbol page**
* `genes/templates/genes/view_gene_symbol.html:141-142` — clones `.gene-info-container` into `#summary-gene-icons-container`. Already dead; nothing in the codebase produces that element.

**Static**
* `variantgrid/static_files/default_static/icons/alt_haplotype.png`
* `variantgrid/static_files/default_static/icons/pseudogene.png`
* `variantgrid/static_files/default_static/icons/triplet_repeat.png`
* `variantgrid/static_files/default_static/css/global.scss:2702` — `.alt-haplotype-icon`, `.pseudogene-icon`, `.triplet-repeat-icon` (and the generated `global.css`)

**Tests**
* `genes/tests/test_urls.py:158` — `api_gene_info`, plus the commented-out `api_batch_gene_info` at `:244`

**Docs**
* `variantgrid_docs` `genes/genegrid.md` — the "Gene Info" section and `images/gene_info/*.png`

### Similarly-named things to keep

Several unrelated things share the name and should survive:

* `genes/templates/genes/gene_symbol_info_tab.html` — `addGeneInfoSummary`, `geneInfoTab`. This is the PanelApp gene summary.
* `variantopedia/templates/variantopedia/variant_details.html:237-244` — `#gene-info` / `gene_symbol_info_tab`.
* `annotation/templates/annotation/tags/variant_transcript_select.html:102,106` — a different `setGeneInfo(geneId, geneSymbol)`.
* `genes/cached_web_resource/refseq.py` and `import_ncbi_gene_info` — NCBI's `Homo_sapiens.gene_info.gz`.

## Follow-up: GIAB challenging medically relevant genes as a CachedWebResource

The pseudogene category is worth keeping in spirit, but "has a pseudogene" is a proxy for what curators actually need to know: short reads mismap here, so treat calls with suspicion.

GIAB's Challenging Medically Relevant Genes (Wagner et al. 2022) measures that directly — roughly 395 genes, published as a gene list and BED, versioned, citable and maintained. It picks up PMS2, SMN1/2, CYP21A2, HBA1/2, STRC, NEB etc. without anyone hand-maintaining a spreadsheet. SA Path diagnostics already consume GIAB-style difficult regions in their pipeline, so it is consistent with how the lab already reasons about this.

Rather than a hand-wired gene list, this should come in as a `CachedWebResource` — the same pattern as MANE, HGNC, gnomAD gene constraint and PanelApp panels (`genes/tasks/cached_web_resource_tasks.py`, wired via `genes/signals/manual_signals.py` and `apps.GenesConfig`). That makes it self-installing and versioned, so there is no manual admin step to document.

Should be raised as its own issue once the retirement above is done.

**Comment by davmlaw** [comment]
The other thing is perhaps we'd be better off doing this in cdot.

It's generated in diagnostics via:

**hap**

contains 'hap' or 'Un_' or '_random'

**

Gene list of 30 genes called TripletRepeatDisorders.txt - need to chase up further info

**Pseudogenes**

Gene list of 3,517 genes,  called Pseudogenes.txt need to ask diagnostics about further info

**Comment by davmlaw** [comment]
Shariant testing not required

Tested VG test - gene grid still functions ok
